javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -355,45 +354,53 @@ public static ExecutableElement elementFromUse(NewClassTree node) {
   }
 
   /**
-   * Determines the symbol for a constructor given an invocation via {@code new}.
+   * Returns the super constructor invoked by {@code newClassTree}.
+   *
+   * <p>If {@code newClassTree} is an anonymous class, then this method returns the super
```

Comment:
This is a bit imprecise: `newClassTree` is always `new ...`.  It might be creating an anonymous class, however.
Same comment applies below for "not an anonymous class".